components/metris/internal/provider/azure/azure.go

```diff
@@ -28,6 +34,13 @@ var (
 
 // NewAzureProvider returns a new Azure provider.
 func NewAzureProvider(config *provider.Config) provider.Provider {
+	// enable azure go-autorest tracing
+	if tracing.IsEnabled() {
+		if err := opencensus.Enable(); err != nil {
+			panic(err)
```

Comment:
Does it make sense to use `log.Panic(err)`?

Reply:
no it does not, I will change that.